policy.go

```diff
@@ -31,20 +31,102 @@ type (
 	Bucket struct {
 		Key      string
 		Value    string
-		nodes    Uint32Slice
+		nodes    Nodes
 		children []Bucket
 	}
 
-	// Uint32Slice is generic type for more convenient sorting.
-	Uint32Slice []uint32
+	// Node type represents single graph leaf with index N and weight W
+	Node struct {
+		N uint32
+		W uint64
+	}
+
+	// Nodes represents slice of graph leafs
+	Nodes []Node
```

Comment:
I propose to replace it to protobuf, I think it's more efficient

and remove methods Write / Read..

@AlexVanin what you think?

Reply:
Seems good. I think in next PR we can define whole `Bucket` structure in protobuf file?

Reply:
kk, I'll create an issue for that

Reply:
https://github.com/nspcc-dev/netmap/issues/24

Reply:
@im-kulikov I tried to use protobuf to define `Node` structure but it was not very effective. Protobuf does not define `Read()` and `Write()` functions, but `Bucket` marshaling is based on these operations. 

Actually we do not use `netmap` marshaling in our code since our `netmap` is built on options. `NodeOption`s are defined with protobuf. 

I propose discuss protobuf structure of `Bucket` in #24 and do not accept changes in this PR.

Reply:
Yeah, that’s why I create issue and resolve this discussion